samples/subsys/usb/cdc_acm/prj.conf

```diff
@@ -10,11 +10,3 @@ CONFIG_USB_DEVICE_LOG_LEVEL_ERR=y
 CONFIG_SERIAL=y
 CONFIG_UART_INTERRUPT_DRIVEN=y
 CONFIG_UART_LINE_CTRL=y
-
-CONFIG_USB_COMPOSITE_DEVICE=n
```

Comment:
That was useful for composite testing, I would be happy if it stays or please add overlay for composite configuration.

Reply:
Do you mean you change `=n` to `=y` when testing? Maybe this can be kept better in private repository. I added test section with composite enabled in the last patch.

Reply:
I can create overlay, this is better way.